kuzzleio/sdk-go
# Launch Java SDK tests on CodePipeline

## What does this PR do ?
Add Continuous Integration with CodePipeline running Java E2E and Unit Tests

### How should this be manually tested?

- Choose .codepipeline/build*.yml matching your machine architecture and replay all the steps.
- Go check CodePipeline build status and logs on AWS

### Other changes

- Fix paths in `internal/wrappers/features/run*.sh` for CI environment.
- Update images stored at https://hub.docker.com/r/kuzzleio/sdk-cross to contain a tests runner script


### Issue fixed
fix kuzzleio/kuzzle-backlog#232
